Clean up event bus code (#4344)
kyma-project/kyma: components/event-bus/README.md

Currently, Event Bus makes the following three Docker images available to the `kyma core` Helm chart:

- event-bus-publish-knative
- event-bus-subscription-controller-knative

There are also end-to-end test Docker images to use as `helm tests`. See [the tests in the `event-bus` directory](https://github.com/kyma-project/kyma/tree/master/tests/event-bus) for more details.

## Development

> The two binaries of `Event Bus` reside under `cmd/event-bus-XXXX` "e.g. `cmd/event-bus-publish-knative`". They each have a Makefile to build and test the component as well as to create and push a Docker image. The following table explains the various make targets.

Reviewer: ```suggestion
The two Event Bus binaries reside in the [`event-bus/cmd`](https://github.com/kyma-project/kyma/tree/master/components/event-bus/cmd) directory.  Each of them has a Makefile to build and test the component as well as to create and push the Docker image. The following table explains various `make` targets.
```
